Plugins/Sycl/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+#find_library(CUDART_LIBRARY cudart ${CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES})   
+
+# This module needs at least CMake 3.13.
+cmake_minimum_required( VERSION 3.13 )
+
+# We use CMake's built-in modules, used to look for SYCL capabilities.
+include( CheckIncludeFileCXX )
+include( CheckCXXSourceCompiles )
+include( CMakeParseArguments )
+set( CMAKE_REQUIRED_QUIET TRUE )
+
+# set(SYCL_DIR ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Apart from the `cmake_minimum_required(...)` line nothing before (and including) this line should be necessary for this plugin.